src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure/AzureClientExtensions.cs

```diff
@@ -356,6 +356,8 @@ public static class AzureClientExtensions
             }
             return true;
         }
+   
+/*
```

Comment:
@hovsepm the plan was to keep this code for two reason:
1) makes it easy to code review the old logic and the new
2) Also for any unforeseen bugs, it's easy to test it against the old logic by uncommenting the code.
3) In next few weeks, once we publish this nuget package, the commented code will be removed.